bip-0181.md

```diff
@@ -0,0 +1,647 @@
+```
+  BIP: 181
+  Layer: Peer Services
+  Title: Utreexo Accumulator Specification
+  Author: Tadge Dryja <rx@awsomnet.org>
+          Calvin Kim <bip@calvinkim.info>
+          Davidson Souza <bip@dlsouza.dev>
+  Comments-Summary: No comments yet.
+  Comments-URI: https://github.com/bitcoin/bips/wiki/Comments:BIP-0181
+  Status: Draft
+  Type: Standards Track
+  Created: 2025-06-18
+  License: BSD-3-Clause
+```
+
+## Abstract
+
+This BIP describes the Utreexo accumulator and its operations. It lays down how to update the
+accumulator as well as how to generate and verify inclusion proofs for elements in the accumulator.
+
+## Motivation
+
+The Bitcoin network is composed of a set of nodes that validate blocks and
+transactions as they are received. These nodes need to keep track of the current state of the network in order to
+fulfill their role. Most importantly, they must maintain a record of all coins that
+have been created but not yet spent, a collection known as the UTXO set.
+
+This set is typically stored in a database that must be accessed frequently and cannot
+be pruned. As a result, the cost of running a node is directly tied to the size
+of the UTXO set. Since it can grow indefinitely, bounded only by block size, it represents a
+long-term scalability concern.
+
+Utreexo is a dynamic accumulator that enables the UTXO set to be represented in just a few kilobytes,
+by requiring peers to provide additional proof data to verify the inclusion of a UTXO in the
+accumulator. This allows for the construction of extremely lightweight nodes capable of performing
+the same validation as a full node, without the need to store the entire UTXO set.
+
+This BIP defines how the Utreexo accumulator works, defining the data structure and algorithms used to
+maintain the accumulator, as well as how to generate and verify inclusion proofs for elements in the accumulator.
+It does not define how the accumulator is used in the Bitcoin protocol, but rather provides a foundation for future
+BIPs that will define how to integrate Utreexo into Bitcoin validation and P2P network protocol.
+
+## License
+
+This document is licensed under the BSD-3-Clause license.
+
+## Preliminaries
+
+An accumulator is a cryptographic data structure that allows for the compact representation of a set,
+enabling efficient membership proofs without requiring storage of the entire set. In the context of Utreexo,
+the accumulator tracks the current set of unspent transaction outputs (UTXOs).
+
+The Utreexo accumulator is based on an append-only Merkle tree design introduced in [^1],
+which provides logarithmic-sized inclusion proofs. Utreexo extends this design to support dynamic updates,
+specifically enabling deletions from the set—a requirement for tracking UTXO spends in Bitcoin.
```

Comment:
Parsing through the linked paper it claimed that the accumulator defined there is sound and strong.

With the extension here to make that accumulator dynamic, I suppose it is still correct, sound and strong?
Perhaps link to some resource on where that was explicitly studied